src/app/static/src/tests/components/reports/runReportInline.test.js

```diff
@@ -27,6 +31,16 @@ describe("runReportInline", () => {
         const wrapper = getWrapper()
         expect(wrapper.find("div a").attributes("href")).toBe("http://app/run-report?report-name=report-data")
     })
+
+    it(`clicking link clears run report key and tab from session`, () => {
+        Storage.prototype.setItem = jest.fn();
+        const spySetStorage = jest.spyOn(Storage.prototype, 'setItem').mock;
+        const wrapper = getWrapper()
+        const link = wrapper.find("div a")
+        link.trigger("click")
+        expect(spySetStorage.calls[0][0]).toBe("selectedRunningReportTab");
```

Comment:
For some reason, the "selectedRunningReportKey" is not appearing in the call. Perhaps because I'm setting it to an empty string when it is already an empty string.

Reply:
This is due to https://github.com/vimc/orderly-web/blob/mrc-2344/src/app/static/src/js/utils/session.js#L10 - empty string is falsey, so results in key being removed. Is that the intended behaviour? Is so then you should check that `removeItem` is call on mock.

Reply:
The line of code I was trying to test has been removed now so no further additions to the test are necessary but that's good to know for future reference.